acl/acl.go

```diff
@@ -75,6 +75,11 @@ type ACL struct {
 	operator string
 	quota    string
 	plugin   string
+
+	// The attribute below is avirtual policy that we never expose directly to
+	// the end user
+	server   string
+	isLeader bool
```

Comment:
Just checking my logic that currently this value will always be false, and this is expected?